samples/Store/Domain/Favorites.fs

```diff
@@ -82,7 +82,8 @@ type Service internal (resolve: ClientId -> Equinox.Decider<Events.Event, Fold.S
     // NOTE not a real world example - used for an integration test; TODO get a better example where it's actually relevant
     member _.UnfavoriteWithPostVersion(clientId, sku) =
         let decider = resolve clientId
-        decider.TransactEx((fun c -> (), decideUnfavorite sku c.State), fun () c -> c.Version)
+        let mapResult () (c: Equinox.ISyncContext<_>) = c.Version
```

Comment:
I doubt this is an obvious solution to anyone new to F# - the error message does mention 'a type annotation'